### DISCUSSION TITLE
Nylo localization RTL issue occurred when we select Arabic or other RTL language

### DISCUSSION THREAD

**Discussion by yagnik1121** [nylo-core/nylo#109] [Q&A] [answered]
One issue encountered with Nylo Framework's internationalization functionality involves the incorrect alignment of Arabic text upon changing the language. Specifically, while using the "change language" function, the Arabic text does not align to the right side as expected. However, upon performing a hot reload, the text aligns correctly. This inconsistency poses a challenge in ensuring proper display of Arabic content, which typically requires right-to-left alignment. Despite the temporary resolution through hot reloading, this issue highlights a need for a more reliable and consistent solution to ensure accurate text alignment in Arabic language settings

**Comment by agordn52** [comment]
Hi @yagnik1121,

Thanks for reporting this, I'll take a look at my earliest convenience to see if this is an issue with Nylo.

**Reply by agordn52** [reply, marked as answer]
Hi @yagnik1121,

You can change the behavior by modifying the `bootstrap/app.dart` file.

Go to the `build` method and update the `locale` argument like in the below example.

``` dart
...
builder: (context, Locale locale, _) => MaterialApp(
              navigatorKey: navigatorKey,
              locale: NyLocalization.instance.locale, // Change `locale` to use `NyLocalization.instance.locale`
```

Hope that helps.
